tests/HeaderTest.php

```diff
@@ -0,0 +1,1050 @@
+<?php
+
+use Lmc\HttpConstants\Header;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @coversDefaultClass \Lmc\HttpConstants\Header
+ */
+final class HeaderTest extends TestCase
```

Comment:
I'd rather have one test for all headers with provider, something like
```
/**
 * @test
 * @dataProvider provideHeaders
*/
public function shouldMatchHeaderKey(string $header, string $expectedKey)
{
    $this->assertSame($expectedKey, $header);
}

public function provideHeaders(): array
{
    return [
        Header::A_IM => [Header::A_IM, 'A-IM'],
        ...
    ];
}
```
This should shorten the test and make it more obvious, readable and extensible.
_I'm not sure about `key` word though_

---
I'd also add test for checking whether all constants are covered.

```
/** @test */
public function shouldCoverAllConstants()
{
    $headersClass = new \ReflectionClass(Header::class);

    $this->assertCount(count($headersClass->getConstants()), $this->provideHeaders());
}
```
:point_up: this could be even smarter, if you wish, but I think at least this kind of test should be there, so it would fail if you add new header.

Reply:
Tests have been updated